lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/mv/XwikiController.java

```diff
@@ -102,6 +104,17 @@ public synchronized void updateBridgeInterface (BridgeInterface bi){
 	@Override
 	public void notifyVerification(String verificationProcessId)
 			throws LpRestException {
+
+	    VerificationStatus status = this.bridge.getVerificationStatus(verificationProcessId);
+	    //TODO: show the status.getStatus() somewhere in the wiki for the verification with id verificationProcessId?
+	    
+	    if(!status.getStatus().equals(StatusType.COMPLETED)){
+	        //A severe error occurred in the verification phase that is terminated unexpectedly
+	        //The verification results in this case is null. If this.bridge.getVerificationResult will be called an exception will rise up
+	        //Notify somewhere the status of the verification id? or obtain the modelSet Id from the verification id and notify the verification status of the modelset id?
+	        return;
+	    }
+	    
```

Comment:
No the part is completed. When the IF happen means that the verification process is terminated with an error (in an unexpected way like process brutally killed) so you can not evaluate the verification result because there is no result so it simply terminate the function.